contracts/distribution/ZNSEthRegistrar.sol

```diff
@@ -111,60 +108,56 @@ contract ZNSEthRegistrar is AccessControlled, IZNSEthRegistrar {
         emit DomainRevoked(domainHash, msg.sender);
     }
 
-    function reclaimDomain(bytes32 domainHash)
-    external
-    override
+    function reclaimDomain(bytes32 domainHash) external override
     onlyTokenOwner(domainHash)
```

Comment:
if we nest, let's nest everything. here only modifier is nested. let's keep to the recommended syntax for Solidity - revert back to previous or put everything in one line.